core/src/main/java/org/mule/runtime/core/internal/processor/strategy/AbstractProcessingStrategy.java

```diff
@@ -144,7 +144,7 @@ public final BackPressureReason emit(CoreEvent event) {
       long remainingCapacity = fluxSink.requestedFromDownstream();
       if (remainingCapacity == 0) {
         return EVENTS_ACCUMULATED;
-      } else if (remainingCapacity > (bufferSize > CORES * 4 ? CORES : 0)) {
+      } else if (remainingCapacity > (bufferSize > CORES * 2 ? CORES : 0)) {
```

Comment:
weren't you going to actually increase the size of this buffer to reduce the error rate?

Reply:
no need to do that, this remained of a test i made, will revert it